engine/src/main/java/org/hibernate/search/indexes/spi/IndexManager.java

```diff
@@ -119,10 +117,4 @@
 	 * To optimize the underlying index. Some implementations might ignore the request, if it doesn't apply to them.
 	 */
 	void optimize();
-
-	/**
-	 * @return the Serializer implementation used for this IndexManager
-	 */
-	LuceneWorkSerializer getSerializer();
```

Comment:
Made a service, as it's not needed for users of the `IndexManager` contract. It's only needed internally within remoting  `BackendQueueProcessor` implementations. They also could manage the serializer themselves, but then I thought that's what services are for in the end.